devops/actions/run-tests/cts/action.yml

```diff
@@ -0,0 +1,159 @@
+name: 'Run SYCL CTS tests'
```

Comment:
This change is meaningless without the part in `.github/workflows/sycl-linux-run-tests.yml`, yet I don't see that here. Can you clarify?

Reply:
Yep, the thing is that we use workflows from `sycl` branch in `sycl-rel nightly`, but the `devops` dir from `sycl-rel` branch. I assume that may be we should use `devops` from `sycl` branch for both builds, but the build may be somehow not compatible with new dependencies / cts tests and so on.

Reply:
@aelovikov-intel any other comments?

Reply:
Can we use `devops/actions` from `sycl` branch? And the configs/deps from `sycl-rel`.